app/models/concerns/account_settings.rb

```diff
@@ -44,6 +44,7 @@ module AccountSettings
     setting :google_scholarly_work_types, type: 'array', disabled: true
     setting :geonames_username, type: 'string', default: ''
     setting :gtm_id, type: 'string'
+    setting :hidden_index_fields, type: 'string', default: 'title'
```

Comment:
is title really a good one to hide by default?

Reply:
Hiding title mirrors the state before this change:
```ruby
config.add_index_field 'title_tesim', label: "Title", itemprop: 'name', if: false
```
Otherwise, title would show up twice in the search results; once as the header for the result, and once in its list of metadata

Reply:
great, this is gtg then